src/ansys/fluent/core/launcher/container_launcher.py

```diff
@@ -80,12 +82,14 @@ def __init__(
         product_version : FluentVersion, optional
             Version of Ansys Fluent to launch. Use ``FluentVersion.v241`` for 2024 R1.
             The default is ``None``, in which case the newest installed version is used.
-        version : str, optional
+        dimension : Dimension or str, optional
             Geometric dimensionality of the Fluent simulation. The default is ``None``,
-            in which case ``"3d"`` is used. Options are ``"3d"`` and ``"2d"``.
-        precision : str, optional
+            in which case ``"three"`` is used. Options are either the values of the ``Dimension``
+            enum or any of ``"three"`` and ``"two"``.
+        precision : Precision or str, optional
             Floating point precision. The default is ``None``, in which case ``"double"``
-            is used. Options are ``"double"`` and ``"single"``.
+            is used. Options are either the values of the ``Precision`` enum or any
+            of ``"double"`` and ``"single"``.
```

Comment:
I would make a similar point here. The most correct usage is `Precision.DOUBLE` etc. I would much prefer to see it documented that way.

Reply:
ok, I agree. Will update the doc accordingly.
Thanks.

Reply:
Done, Thank you.